Deployment_Strategies/Blue-green-deployment/online-shop-without-footer-blue-deployment.yaml

```
@@ -0,0 +1,44 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: online-shop-blue-deployment-service
+  namespace: blue-green-ns
+spec:
+  selector:
+    app: online-shop-blue
+  type: NodePort
+  ports:
+    - protocol: TCP
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)